packages/office-ui-fabric-react/src/components/Facepile/examples/Facepile.Basic.Example.tsx

```diff
@@ -26,13 +26,15 @@ export class FacepileBasicExample extends React.Component<{}, IFacepileBasicExam
 
   public render(): JSX.Element {
     const { numberOfFaces, personaSize } = this.state;
+
     const facepileProps: IFacepileProps = {
       personaSize: personaSize,
       personas: facepilePersonas.slice(0, numberOfFaces),
       overflowPersonas: facepilePersonas.slice(numberOfFaces),
       getPersonaProps: (persona: IFacepilePersona) => {
         return {
-          imageShouldFadeIn: this.state.imagesFadeIn
+          imageShouldFadeIn: this.state.imagesFadeIn,
+          presence: this._personaPresence(persona.personaName)
```

Comment:
You might add a note that typically the persona presence would be included when generating each facepile persona (probably when processing data from a server), but here we're assigning an arbitrary presence to each persona based on its name for demonstration purposes.